openhtf/__init__.py

```diff
@@ -334,19 +364,19 @@ class PhaseInfo(mutablerecords.Record(
 
   @classmethod
   def WrapOrCopy(cls, func):
```

Comment:
hm, i'll defer that to a separate change, could be as simple as an alias in `names.py`